extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -471,13 +476,40 @@ export class AmpStoryPage extends AMP.BaseElement {
   }
 
   /**
-   * Finds embedded components in page and prepares them for their expanded view
-   * animation.
+   * Finds embedded components in page and prepares them.
    * @param {boolean=} forceResize
    * @private
    */
   findAndPrepareEmbeddedComponents_(forceResize = false) {
-    scopedQuerySelectorAll(this.element, EMBEDDED_COMPONENTS_SELECTORS)
+    this.addClickShieldToEmbeds_();
+    this.resizeInteractiveEmbeddedComponents_(forceResize);
+  }
+
+  /**
+   * Adds a pseudo element on top of the embed to block clicks from going into
+   * the iframe.
+   * @private
+   */
+  addClickShieldToEmbeds_() {
+    const els = [];
+    scopedQuerySelectorAll(this.element, EMBEDDED_COMPONENTS_SELECTORS).forEach(
+        el => els.push(el));
```

Comment:
Do you need to create this extra array, or can you just loop through the result of `scopedQuerySelectorAll()`?